runtime/calamari/src/lib.rs

```diff
@@ -615,7 +616,6 @@ impl pallet_session::Config for Runtime {
 	type SessionHandler =
 		<opaque::SessionKeys as sp_runtime::traits::OpaqueKeys>::KeyTypeIdProviders;
 	type Keys = opaque::SessionKeys;
-	type DisabledValidatorsThreshold = DisabledValidatorsThreshold;
```

Comment:
@stechu @Dengjianping 
You can review https://github.com/paritytech/substrate/pull/9448 for this API change.